scripts/dfu.py

```diff
@@ -112,7 +112,7 @@ def clear_status(self):
 			usb.RECIP_INTERFACE, DFU_CLRSTATUS,
 			"", index=0)
 
-	def get_state(self):
+		def get_state(self):
```

Comment:
This seems like a separate bug fix unrelated to the new platform addition?

Reply:
Ah, yes, hadn't even spotted that got in to this - we'll rework to remove that tomorrow.

Reply:
Sounds good :D

Reply:
As discussed on "sideband" I resolved this. :D